plasma_framework/contracts/mocks/utils/QuarantineMock.sol

```diff
@@ -0,0 +1,33 @@
+pragma solidity ^0.5.0;
+
+import "../../src/framework/utils/Quarantine.sol";
+
+contract QuarantineRegistryMock is Quarantine {
+    mapping(uint256 => address) private _contracts;
+
+    constructor(uint256 _period, uint256 _initialImmuneCount)
+    public
+    Quarantine(_period, _initialImmuneCount)
+    {}
+
+    function registerContract(uint256 _contractId, address _contractAddress) public {
```

Comment:
It took me quite a while to understand why `register` in here. And then I realize it is probably mimicking where `Quarantine` is used (VaultRegistry). I feel `Quarantine` does not seems to be designed to be used only in registry though. Personally I would just call `Quarantine.quarantine` directly in test instead of having a registry to call that.

With above mind set, I would just have `QuarantinedContractMock` and probably only need to extend the `test` function.